app/styles/main.less

```diff
@@ -81,6 +81,9 @@
 @import "@{pf-less-path}/type.less";
 @import "@{pf-less-path}/vertical-nav.less";
 
+/* Angular Key Value Editor */
+@import "../../bower_components/angular-key-value-editor/src/less/angular-key-value-editor.less";
```

Comment:
would prefer to keep all the relative path calculations in variables at the very top of the file, create a new var for bower-components-path